src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -4,19 +4,30 @@ export default {
     step1_label: T('What and Where'),
 
     exist_replication_placeholder: T('Load Previous Replication Task'),
-    exist_replication_tooltip: T(''),
+    exist_replication_tooltip: T('Fill the wizard with the settings from \
+ a saved replication.'),
 
     source_datasets_from_placeholder: T('Source Datasets'),
-    source_datasets_from_tooltip: T(''),
+    source_datasets_from_tooltip: T('Datasets that have snapshots to \
+ replicate. Define the location of the source datasets, then select \
+ datasets for replication. Choosing <i>On a Different System</i> \
+ requires configuring an SSH connection to the remote system. '),
 
-    target_dataset_from_placeholder: T('Destination Datasets'),
-    target_dataset_from_tooltip: T(''),
+    target_dataset_from_placeholder: T('Destination Dataset'),
+    target_dataset_from_tooltip: T('Dataset to store replicated \
+ snapshots. Define the location of the destination dataset, then select \
```

Comment:
Again with "define"... AFAIK, users are choosing a target dataset, so the location is just part of that choice.  So maybe this can be simplified to "Choose the dataset where replicated snapshots are stored."

Reply:
The help text was broken for some of the sub-fields to this drop-down, so I attempted to combine the flow of the selections into one paragraph. I've since fixed the help text links and will split help text between all 3-4 fields.